easybuild/easyconfigs/s/Salmon/Salmon-1.2.1-gompi-2020a.eb

```diff
@@ -0,0 +1,59 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     GLPv2
+#
+# Notes::
+##
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+# Thanks to people like Simon Brandord Mikael Oehmann and Kenneth Hoste
+# for their help with the ICE
+
+easyblock = 'CMakeMake'
+
+name = 'Salmon'
+version = '1.2.1'
+
+homepage = 'https://github.com/COMBINE-lab/salmon'
+description = """Salmon is a wicked-fast program to produce a highly-accurate,
+ transcript-level quantification estimates from RNA-seq data."""
+
+toolchain = {'name': 'gompi', 'version': '2020a'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+github_account = 'COMBINE-lab'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['10a58a22cb5b3924df3bf78be9cb4b83dce4f6a61e4c6a6e37f3a56ab8ac166f']
+
+builddependencies = [
+    ('CMake', '3.16.4'),
+    ('pkg-config', '0.29.2'),
+    ('jemalloc', '5.2.1'),
+]
+
+dependencies = [
+    ('Boost', '1.72.0'),
+    ('tbb', '2020.1'),
+]
+
+configopts = '-DJEMALLOC_ROOT=$EBROOTJEMALLOC -DNO_IPO=1'
```

Comment:
@sassy-crick I think it makes sense to clarify with a comment above this line what the `-DNO_IPO=1` does, and why it's there. Something like this:

```python
# disable link-time optimizations (-flto) because it triggers a segfault/internal compiler error (ICE) with GCC 9.3.0
configopts = '-DJEMALLOC_ROOT=$EBROOTJEMALLOC -DNO_IPO=1'
```

Reply:
@boegel Done. I thought of it as well. I also expanded on it and pointed out the problem might be gone with a newer version of GCC or Salmon. I will upload the corrected version once I know what to do with the dependency question above.

Reply:
In light of a newer version ([PR#10997](https://github.com/easybuilders/easybuild-easyconfigs/pull/10997)) I will close this one here as it is of limited value. Thanks for the help!